### PR TITLE
FH-3648 - Change the cmd user to follow the V3 standard and add unit

### PR DESCRIPTION
**Pull request by camilamacedo86**
This PR is a fix of: https://github.com/feedhenry/fh-fhc/pull/346

### CODE REVIEW

**Comment by camilamacedo86** [comment]
@witmicko this one will be easer if can be review/approved by you. Could you check?

File: lib/cmd/fh3/user.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/* globals i18n */
+var ini = require("../../utils/ini");
+
+module.exports = {
+  'desc' : i18n._('User information'),
+  'examples' :
+    [{
+      cmd : 'fhc user',
+      desc : i18n._('Show the information of the logged user into the target domain')
```
Comment (wtrocki) [inline lib/cmd/fh3/user.js]: into => for

**Comment by wtrocki** [comment]
See also comment in: https://github.com/feedhenry/fh-fhc/pull/346

File: lib/cmd/fh3/user.js (hunk 1 above)
Comment (camilamacedo86) [inline lib/cmd/fh3/user.js]: 👍 I'll change the text 'into' to 'for'

**Comment by camilamacedo86** [comment]
@wtrocki,

Could you help understand what you think is missing here relate the comment #346 ?

Because I understood that was the following code after make the login with success which is in the commit fix : https://github.com/feedhenry/fh-fhc/pull/353/files#diff-cfd7326ca1a68e6c504f3e77e58bffd0R38.

**Comment by wtrocki** [comment]
After this change ini will be missing user pemissions. This commad was used in login process. 
We can verify this using following step:
- Remove fhc config from home
- fh login username
- Verify if file have user permissions.
If they are there I will approve this PR :P

**Comment by camilamacedo86** [comment]
Hi @wtrocki,

Yes the permissions are saved. Following the local test with this branch. 

screen shot 2017-06-09 at 8 09 26 pm

screen shot 2017-06-09 at 8 08 51 pm

Following the steps performed to test it. 

`Camilas-MBP:~ cmacedo$ rm -rf .fhcrc  .fhctargets `

```
Camilas-MBP:fh-fhc cmacedo$ ./bin/fhc.js target https://support.us.feedhenry.com/
Successfully targeted https://support.us.feedhenry.com/ - now, log in.
fhc login <username> <password>
You are prompted for username/password if none are provided.
Camilas-MBP:fh-fhc cmacedo$ ./bin/fhc.js login
Username: cmacedo@redhat.com
Password: 

Camilas-MBP:fh-fhc cmacedo$ 
```

```
Camilas-MBP:~ cmacedo$ cat .fhcrc 
fhclatest = {"version":"2.17.5-685","is":true,"ts":1497049680948,"current":"2.17.5-BUILD-NUMBER"}
showlevel = 2
fhversion = 3
feedhenry = https://support.us.feedhenry.com/
username = cmacedo@redhat.com
cookie = yxn3xybpft5wtd5g5xqum4xs
domain = support
perms = cluster/reseller/customer/domain/project/client-apps/source-code:read,cluster/reseller/customer/domain/admin/app-store/device:write,cluster/reseller/customer/domain/project/cloud-apps/endpoint:write,cluster/reseller/customer/domain/project/analytics:write,cluster/reseller:write,cluster/reseller/customer/domain/project/cloud-apps/data-browser:read,cluster/reseller/customer/domain/admin/app-store/store-item-group:write,cluster/reseller/customer/domain/analytics:read,cluster/reseller/customer/domain/project/lifecycle:write,cluster/reseller/customer/domain/project/cloud-apps/cloud-deployment:read,cluster/reseller/customer/domain/templates:read,cluster/reseller/user:write,cluster/reseller/customer/domain/service:read,cluster/reseller/customer/domain/service/documentation:read,cluster/reseller/customer/domain/project/cloud-apps/environment-variable:read,cluster/reseller/customer/domain/drag-and-drop-app/theme:write,cluster/reseller/customer/domain/service/cloud-deployment:write,cluster/reseller/customer/domain/service/documentation:write,cluster/reseller/customer/domain/project/client-apps/event-log:write,cluster/reseller/customer/domain/project/cloud-apps/notification:write,cluster/reseller/customer/domain/admin/mbaas-target:read,cluster/reseller/customer/domain/project/legacy-app-key:read,cluster/reseller/customer/domain/admin/user:write,cluster/reseller/customer/domain/project/cloud-apps/notification:read,cluster/reseller/customer/domain/project:read,cluster/reseller/user/credential-bundle:read,cluster/reseller/customer/domain/project/cloud-apps/event-log:write,cluster/reseller/user/credential:read,cluster/reseller/customer/domain/admin/auth-policy:write,cluster/reseller/team:read,cluster/reseller/customer/domain/project/analytics:read,cluster/reseller/customer/domain/admin/mbaas-target:write,cluster/reseller/customer/domain/project/client-apps/credentials:read,cluster/reseller/customer/domain/project/cloud-apps/api-key:read,cluster/reseller/customer/team:write,cluster/reseller/customer/domain/service/cloud-apps:read,cluster/reseller/customer/domain/analytics:write,cluster/reseller/customer/domain/project/client-apps/log:write,cluster/reseller/customer/domain/admin/app-store/store-item/store-item-screenshot:read,cluster/reseller/customer/domain/project/cloud-apps/data-browser:write,cluster/reseller/customer/domain/project/client-apps/environment-variable:read,cluster/reseller/customer/domain/service/stat:read,cluster/reseller/customer/domain/service/data-browser:write,cluster/reseller/customer/domain/project/client-apps/endpoint:write,cluster/reseller/customer/domain/project/cloud-apps/notification-log:read,cluster/reseller/customer/domain/project/client-apps/endpoint:read,cluster/reseller/customer/domain/service/cloud-deployment:read,cluster/reseller/team:write,cluster/reseller/customer/user/credential:write,cluster/reseller/customer/domain/project/client-apps/data-browser:write,cluster/reseller/user/credential-bundle:write,cluster/reseller/customer/domain/project/cloud-apps/documentation:read,cluster/reseller/customer/domain/project/cloud-apps/documentation:write,cluster/reseller/customer/domain/templates:write,cluster/reseller/customer/domain/service/notification:write,cluster/reseller/customer/domain/project/client-apps/notification-log:read,cluster/reseller/customer/domain/project/client-apps/analytics:read,cluster/reseller/customer/domain/project/connection:write,cluster/reseller/customer/domain/project/client-apps/configuration:read,cluster/reseller/customer/domain/project/client-apps:read,cluster/reseller/customer/domain/service/cloud-apps/log:read,cluster/reseller/customer/domain/project/client-apps/notification-log:write,cluster/reseller/customer:read,cluster/reseller/customer/domain/project/legacy-text-file:read,cluster/reseller/customer/domain/repo:write,cluster/reseller/customer/domain/project/client-apps/configuration:write,cluster/reseller/customer/domain/admin/environment:read,cluster/reseller/customer/domain/service/environment-variable:read,cluster/reseller/customer/domain/service/stat:write,cluster/reseller/customer/domain/project/client-apps/push-notifications:read,cluster/reseller/customer/user/credential-bundle:read,cluster/reseller/customer/domain/drag-and-drop-app:read,cluster/reseller/customer/domain/drag-and-drop-app/form/deploy:write,cluster/reseller/customer/domain/service/environment-variable:write,cluster/reseller/customer/domain/drag-and-drop-app/theme:read,cluster/reseller/customer/user:read,cluster/reseller/customer/user/credential:read,cluster/reseller/customer/domain/project/cloud-apps/endpoint:read,cluster/reseller/customer/domain/project/client-apps/data-browser:read,cluster/reseller/customer/user/api-key:read,cluster/reseller/customer/domain/service/source-code:write,cluster/reseller/customer/domain/drag-and-drop-app/form/submission:write,cluster/reseller/customer/domain/admin/app-store/store-item/store-item-binary/store-item-binary-version:write,cluster/reseller/customer/domain:read,cluster/reseller/customer/domain/project/cloud-apps/cloud-deployment:write,cluster/reseller/customer/domain/admin/app-store/log:write,cluster/reseller/customer/domain/admin/app-store/device:read,cluster/reseller/customer/domain/admin/app-store/store-item/store-item-binary/store-item-binary-version:read,cluster/reseller/customer/domain/project/client-apps:write,cluster/reseller/customer/domain/project/client-apps/api-key:write,cluster/reseller/customer/domain/service/cloud-apps:write,cluster/reseller/customer/domain/admin/app-store/store-item/store-item-binary:read,cluster/reseller/customer/domain/project/client-apps/binary:write,cluster/reseller/customer/domain/drag-and-drop-app/form:write,cluster/reseller/user:read,cluster/reseller/customer/domain/project/client-apps/analytics:write,cluster/reseller/customer/domain/project/lifecycle:read,cluster/reseller/customer:write,cluster/reseller/customer/domain/project/cloud-apps:read,cluster/reseller/user/api-key:read,cluster/reseller/customer/domain/project/client-apps/stat:write,cluster/reseller/customer/domain/admin/app-store/audit-log:read,cluster/reseller/customer/domain/cloud-resources:write,cluster/reseller/customer/domain/admin/auth-policy:read,cluster/reseller/customer/domain/cloud-resources:read,cluster/reseller/customer/domain/project/service:read,cluster/reseller/customer/domain/drag-and-drop-app/form/submission:read,cluster/reseller/customer/domain/service/endpoint:read,cluster/reseller/customer/domain/project/client-apps/repo:read,cluster/reseller/customer/domain/project/legacy-app-key:write,cluster/reseller/customer/domain/project/legacy-binary-file:read,cluster/reseller/customer/domain/project/client-apps/event-log:read,cluster/reseller/customer/domain/admin/app-store/store-item:read,cluster/reseller/customer/domain/service/event-log:read,cluster/reseller/customer/domain/project/cloud-apps/log:read,cluster/reseller/customer/domain/service/cloud-apps/log:write,cluster/reseller/customer/domain/admin:write,cluster/reseller/customer/domain/admin/app-store/store-item/store-item-binary:write,cluster/reseller/customer/domain/project/cloud-apps/analytics:write,cluster/reseller/customer/domain/admin/user:read,cluster/reseller/user/credential:write,cluster/reseller/customer/domain/service/analytics:read,cluster/reseller/customer/domain:write,cluster/reseller/customer/domain/admin/app-store/store-item-group:read,cluster/reseller/customer/domain/project/legacy-text-file:write,cluster/reseller/customer/domain/admin/app-store/store-item:write,cluster/reseller/customer/domain/project/cloud-apps/environment-variable:write,cluster/reseller/customer/domain/drag-and-drop-app/data-source:read,cluster/reseller/customer/domain/service:write,cluster/reseller/customer/domain/project/cloud-apps/stat:write,cluster/reseller/customer/domain/admin/app-store/store-item/store-item-screenshot:write,cluster/reseller/customer/domain/drag-and-drop-app/project:read,cluster/reseller/customer/domain/project/client-apps/documentation:read,cluster/reseller/customer/domain/drag-and-drop-app:write,cluster/reseller/customer/domain/drag-and-drop-app/data-source:write,cluster/reseller/customer/domain/admin/environment:write,cluster/reseller/customer/domain/project/client-apps/repo:write,cluster/reseller/customer/domain/project/cloud-apps/log:write,cluster/reseller/customer/domain/project/cloud-apps/analytics:read,cluster/reseller/customer/domain/admin/deploy-target:read,cluster/reseller/user/api-key:write,cluster/reseller/customer/domain/admin/app-store/log:read,cluster/reseller/customer/domain/project/client-apps/environment-variable:write,cluster/reseller/customer/domain/drag-and-drop-app/form:read,cluster/reseller/customer/domain/project/cloud-resources:read,cluster/reseller/customer/domain/project/cloud-apps/stat:read,cluster/reseller/customer/domain/project/client-apps/stat:read,cluster/reseller/customer/domain/drag-and-drop-app/project:write,cluster/reseller/customer/user/credential-bundle:write,cluster/reseller/customer/user:write,cluster/reseller/customer/team:read,cluster/reseller/customer/domain/service/notification:read,cluster/reseller/customer/domain/project/connection:read,cluster/reseller/customer/domain/service/source-code:read,cluster/reseller/customer/user/api-key:write,cluster/reseller/customer/domain/project/cloud-apps/repo:read,cluster/reseller/customer/domain/service/data-browser:read,cluster/reseller/customer/domain/project/cloud-apps/repo:write,cluster/reseller/customer/domain/project/client-apps/binary:read,cluster/reseller/customer/domain/admin/deploy-target:write,cluster/reseller/customer/domain/project/client-apps/log:read,cluster/reseller/customer/domain/project/client-apps/documentation:write,cluster/reseller/customer/domain/project/client-apps/cloud-deployment:write,cluster/reseller/customer/domain/project/client-apps/api-key:read,cluster/reseller/customer/domain/admin/app-store/audit-log:write,cluster/reseller/customer/domain/service/endpoint:write,cluster/reseller/customer/domain/project/cloud-resources:write,cluster/reseller/customer/domain/service/analytics:write,cluster/reseller/customer/domain/admin:read,cluster/reseller/customer/domain/project/cloud-apps/source-code:write,cluster/reseller:read,cluster/reseller/customer/domain/project/cloud-apps/source-code:read,cluster/reseller/customer/domain/service/event-log:write,cluster/reseller/customer/domain/project/client-apps/notification:read,cluster/reseller/customer/domain/service/preview:read,cluster/reseller/customer/domain/project/client-apps/credentials:write,cluster/reseller/customer/domain/project/client-apps/push-notifications:write,cluster/reseller/customer/domain/project/client-apps/cloud-deployment:read,cluster/reseller/customer/domain/project/legacy-binary-file:write,cluster/reseller/customer/domain/project/client-apps/notification:write,cluster/reseller/customer/domain/project/client-apps/source-code:write,cluster/reseller/customer/domain/project/service:write,cluster/reseller/customer/domain/admin/team:write,cluster/reseller/customer/domain/project/cloud-apps/notification-log:write,cluster/reseller/customer/domain/project/cloud-apps/event-log:read,cluster/reseller/customer/domain/admin/app-store:read,cluster/reseller/customer/domain/repo:read,cluster/resellCamilas-MBP:~ cmacedo$ 
```

Thank your for your collaboration. :-)

File: lib/cmd/common/ngui.js [hunk 2]
```diff
@@ -6,7 +6,6 @@ ngui.usage = "fhc ngui" +
   i18n._("\nfhc ngui disable - disable FH3 for the current user");
```
Comment (wtrocki) [inline lib/cmd/common/ngui.js]: I assume that this will also land in fh3?

Comment (camilamacedo86) [inline lib/cmd/common/ngui.js]: The V2 commands still there as the fhversion. So, all still the same so far :-)

**Pull request merged**